go/rootprogram/issuers.go

```diff
@@ -233,7 +234,7 @@ func (mi *MozIssuers) LoadEnrolledIssuers(filePath string) error {
 		if err != nil {
 			return err
 		}
-		mi.InsertIssuerFromCertAndPem(cert, ei.Pem, nil)
+		mi.InsertIssuerFromCertAndPem(cert, ei.Pem, nil, false)
```

Comment:
Does go have a maybe/option type? It would be nice to ensure we're not accidentally using the value of `usesPartitionedCrls` when we don't load from the ccadb.

Reply:
I updated this so now we pipe the `usesPartitionedCrls` flag through the enrolled.json file for use here.